### PR TITLE
Feature/lineup editor

### PR DESCRIPTION
**Pull request by jarhodes314**
Add lineup editor
Parallelise tests

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -5,18 +5,47 @@ on:
...
+      - name: Checkout
+        uses: actions/checkout@v1
+      # because of "record" and "parallel" parameters
+      # these containers will load balance all found tests among themselves
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          record: true
+          parallel: true
+          group: 'Actions example'
```
Comment (caro401) [inline .github/workflows/test.yml:29]: can haz better group name?

File: hasura/metadata/actions.graphql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (caro401) [inline hasura/metadata/actions.graphql:1]: ??? why does this need blank lines in an empty file?

**Review by caro401 — APPROVED** [review]

File: hasura/metadata/actions.graphql (hunk 2 above)
Comment (jarhodes314) [reply hasura/metadata/actions.graphql:1]: Magic auto generated stuffs

**Comment by jarhodes314** [comment]
This fixed #10

**Pull request merged**